externals/CMakeLists.txt

```
@@ -10,56 +10,57 @@ endif()
 
 # catch
 
-if (DYNARMIC_TESTS AND NOT TARGET catch)
-    add_library(catch INTERFACE)
-    target_include_directories(catch INTERFACE $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/catch/include>)
+if (NOT TARGET Catch2::Catch2)
+    if (DYNARMIC_TESTS)
```

This can probably be simplified as:

```suggestion
if (NOT TARGET Catch2::Catch2 AND DYNARMIC_TESTS)
```

---

Yes, but I was trying to mimic the existing condition for `xbyak` and `Zydis`!